ArduCopter/sensors.cpp

```diff
@@ -478,8 +478,9 @@ void Copter::update_sensor_status_flags(void)
     }
 
     if (copter.failsafe.battery) {
-         control_sensors_health &= ~MAV_SYS_STATUS_SENSOR_BATTERY;                                                                    }
-    
+         control_sensors_health &= ~MAV_SYS_STATUS_SENSOR_BATTERY;
+    }
```

Comment:
How did this compile?

Reply:
That would bit github's interface playing with you :-)

Look *way* over to the right.

Reply:
Ah! I thought I was going crazy 😄